README.md

```diff
@@ -17,17 +17,20 @@ plugins:
 
 ## Configuration
 
+
 ```yaml
 custom:
   globalTables:
   - regions: # list of regions in which you want to set up global tables
       - region-1
       - region-2
-    tableKey: 'TABLE_KEY' # Cloudformation output key name if the table is created as part of same serverless service
-    tableName: 'TABLE_NAME' # if table is not part of the service then specify the table name. If tableKey param exists then tableName is ignored.
+    tableName: 'TABLE_NAME' # *RECOMMENDED* if table is not part of the service then specify the table name. If tableKey param exists then tableName is ignored.
```

Comment:
why are you saying that it's RECOMMENDED not to create the table as part of service? @johndww